From 062e4710c524283c5d11cbf99931b24c6d963f98 Mon Sep 17 00:00:00 2001
From: Marijn Suijten <marijns95@gmail.com>
Date: Thu, 14 Jan 2021 14:11:50 +0100
Subject: [PATCH 17/20] USB-Audio/ALC1220: Bump analog Speaker priority over
 S/PDIF

The `S/PDIF` port does not have any jack sensing, and with a priority
higher than the `Speakers` it will always be selected by default instead
of the `Speakers` even if unplugged.

Swapping the priorities around allows analog `Speakers` to be selected
first, _if_ they are plugged in. Otherwise `S/PDIF` is used.

Signed-off-by: Marijn Suijten <marijns95@gmail.com>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---
 ucm2/USB-Audio/Realtek-ALC1220-VB-Desktop-HiFi.conf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ucm2/USB-Audio/Realtek-ALC1220-VB-Desktop-HiFi.conf b/ucm2/USB-Audio/Realtek-ALC1220-VB-Desktop-HiFi.conf
index 344db8335398..cbf715475e31 100644
--- a/ucm2/USB-Audio/Realtek-ALC1220-VB-Desktop-HiFi.conf
+++ b/ucm2/USB-Audio/Realtek-ALC1220-VB-Desktop-HiFi.conf
@@ -2,7 +2,7 @@ SectionDevice."Speaker" {
 	Comment "Speakers"
 	Value {
 	       PlaybackChannels 8
-	       PlaybackPriority 100
+	       PlaybackPriority 200
 	       PlaybackPCM "hw:${CardId}"
 	       JackControl "Speaker Jack"
 	       PlaybackMixerElem "Speaker"
@@ -22,7 +22,7 @@ SectionDevice."Headphones" {
 SectionDevice."SPDIF" {
 	Comment "S/PDIF Out"
 	Value {
-		PlaybackPriority 200
+		PlaybackPriority 100
 		PlaybackPCM "hw:${CardId},2"
 		PlaybackMixerElem "IEC958"
 	}
-- 
2.26.2

